notifications_utils/recipient_validation/errors.py

```diff
@@ -10,7 +18,42 @@ class InvalidEmailError(InvalidRecipientError):
 
 
 class InvalidPhoneError(InvalidRecipientError):
-    message = "Not a valid phone number"
+    class Codes(StrEnum):
+        DEFAULT = auto()
```

Comment:
`default` as a code doesn't really make sense in isolation, so should change this to something like `invalid_number` and then just use that as the default in the constructor

```suggestion
        INVALID_NUMBER = auto()
```